solidity/dao-congress.sol

```diff
@@ -253,8 +253,9 @@ contract Congress is owned, tokenRecipient {
         onlyMembers public
         returns (uint voteID)
     {
-        Proposal storage p = proposals[proposalNumber];         // Get the proposal
-        require(!p.voted[msg.sender]);         // If has already voted, cancel
+        Proposal storage p = proposals[proposalNumber]; // Get the proposal
+        require(now < p.minExecutionDate);              // If time has left before voting deadline
```

Comment:
You can remove this line

Reply:
I added this line 257 only so people don't execute vote function after the voting time is over.

Reply:
I have removed the line 257 in new commit, please see it.